Mathlib/LinearAlgebra/DirectSum/Finsupp.lean

```diff
@@ -0,0 +1,114 @@
+/-
+Copyright (c) 2019 Johannes Hölzl. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johannes Hölzl
+
+! This file was ported from Lean 3 source module linear_algebra.direct_sum.finsupp
+! leanprover-community/mathlib commit 9b9d125b7be0930f564a68f1d73ace10cf46064d
+! Please do not edit these lines, except to modify the commit id
+! if you have ported upstream changes.
+-/
+import Mathlib.Algebra.DirectSum.Finsupp
+import Mathlib.LinearAlgebra.Finsupp
+import Mathlib.LinearAlgebra.DirectSum.TensorProduct
+
+/-!
+# Results on finitely supported functions.
+
+The tensor product of ι →₀ M and κ →₀ N is linearly equivalent to (ι × κ) →₀ (M ⊗ N).
```

Comment:
```suggestion

The tensor product of `ι →₀ M` and `κ →₀ N` is linearly equivalent to `(ι × κ) →₀ (M ⊗ N)`.
```

Reply:
I know it was already like this in the original file, but we might as well fix it anyway.